R/event.R

```diff
@@ -28,7 +30,7 @@
 #'  
 #' @export
 dyEvent <- function(dygraph, 
-                    date,
+                    x,
```

Comment:
Renaming the parameter could break some user code. It's certainly more coherent to use `x` but I hate to break working code. Maybe have both but print a deprecation warning if `date` is used?